frontend/packages/ceph-storage-plugin/src/components/dashboards/common/breakdown-card/breakdown-body.tsx

```diff
@@ -28,7 +28,7 @@ export const BreakdownCardBody: React.FC<BreakdownBodyProps> = ({
   if (!capacityUsed || !top5MetricsStats.length || hasLoadError) {
     return (
       <div className="text-secondary capacity-breakdown-card--error">
-        {t('dashboard~Not available')}
+        {t('public~Not available')}
```

Comment:
Messages in ceph-storage-plugin should use the `ceph-storage-plugin` namespace.

```suggestion
        {t('ceph-storage-plugin~Not available')}
```